src/locale.en.js

```diff
@@ -66,6 +66,7 @@ export const campsite = {
             river_view: 'River/Creek View',
             secluded: 'Secluded',
             mountain_views: 'Mountain Views',
+            winter_access: 'Accessible in Winter',
```

Comment:
Added option text and icon for winter accessible sites. (Also updated our firebase backend to reflect sites that have this option)